draft-opsarea-rfc5706bis.md

```diff
@@ -884,21 +885,21 @@ author:
 
    2.  Require that all objects be essential for management.
 
-   3.  Consider evidence of current use of the managed protocol, and the perceived utility of objects added to the information model.
+   3.  Consider evidence of current use of the managed protocol, and the perceived utility of objects added to the Information Model.
 
    4.  Exclude objects that can be derived from others in this or
        other information models.
 
    5.  Avoid causing critical sections to be heavily instrumented. A
        guideline is one counter per critical section per layer.
 
-   6.  When defining an information model using  YANG Data Structure Extensions {{?RFC8791}} (thereby keeping it abstract and implementation-agnostic per {{?RFC3444}}) ensure that the information model remains simple, modular, and clear by following the authoring guidelines in {{?I-D.ietf-netmod-rfc8407bis}}.
-  7.  When illustrating the abstract information model, use YANG Tree Diagrams {{?RFC8340}} to provide a simple, standardized, and implementation-neutral model structure.
+   6.  When defining an Information Model using  YANG Data Structure Extensions {{?RFC8791}} (thereby keeping it abstract and implementation-agnostic per {{?RFC3444}}) ensure that the Information Model remains simple, modular, and clear by following the authoring guidelines in {{?I-D.ietf-netmod-rfc8407bis}}.
+  7.  When illustrating the abstract Information Model, use YANG Tree Diagrams {{?RFC8340}} to provide a simple, standardized, and implementation-neutral model structure.
 
 ### YANG Data Model Considerations {#sec-yang-dm}
 
   When considering YANG data models for a new specification, there
-  are multiple types of data models that may be applicable. The
+  are multiple types of Data Models that may be applicable. The
```

Comment:
This goes to Benoît's earlier question about what to do with "YANG data model".  Here, "data models" refers to the types of YANG data models defined below.  Each is a "Data Model" via our terminology.  So, I don't know what is correct here.  Admittedly, I'm bikeshedding a bit.  I think Data Models is probably fine, but I'm commenting in case others feel strongly one way or the other.